live-tests/README.adoc

```diff
@@ -20,6 +20,12 @@ These tests are not currently run automatically (though they are _built_ in CI).
 
 You can run them yourself.  First, deploy Omicron using `a4x2` or one of the hardware test rigs.  In your Omicron workspace, run `cargo xtask live-tests` to build an archive and then follow the instructions:
 
+The live tests require that the system's target blueprint is enabled. This is to avoid a case where the live tests generate blueprints based on a target blueprint that is not current and then wind up making a bunch of unrelated changes to the system. On a fresh system, you will have to enable the target blueprint yourself:
```

Comment:
Sorry, just rereading this in context, the sentence above this is introducing the raw text block below, and now this is sandwiched in between.  This new block seems to belong after "First, deploy Omicron ..." and before "In your Omicron workspace...".